### ISSUE TITLE
Use system DNS instead of Google

### ISSUE THREAD

**Issue by mildred** [rockcavera/nim-ndns#1]
It would be great to use the system DNS instead of relying to Google by default.

**Comment by rockcavera** [comment]
If you're referring to using the DNS server configured in the operating system, I also imagine that would be great.

The biggest challenge for this would be to implement for each operating system. I've thought about doing this, but it never left the field of imagination.

**Comment by rockcavera** [comment]
I don't know if that's what you're talking about, but I started writing code to work on Windows like this:
1) create a global dns client, in which the dns server for consultation is the internet connection; and
2) maps the file C:\Windows\System32\drivers\etc\hosts to have a quick cache for lookup predefined names.

From what I've studied, several operating systems have this hosts file. I'm going to take a look at linux and try to at least make it for these two operating systems, which are the most used.
